### PR TITLE
[FRONT-809] Add Room Modal

### PR DESCRIPTION
**Pull request by its-edmund**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,82 +1,82 @@
 {
-  "name": "the-chat-app",
```
Comment (mondoreale) [inline package.json:2]: If that's prettier then let's revert. Otherwise npm will reformat it back again.

**Review by mondoreale — CHANGES_REQUESTED** [review]
Let's fix the console, `stroke-*` attrs and it's g2g.

**Comment by hpihkala** [comment]
Just to check - what's implemented in this PR? Is this this?
Screenshot 2022-02-16 at 10 26 15

**Comment by mondoreale** [comment]
@hpihkala, that's the one.

**Pull request merged**